contract-diff.sh

```diff
@@ -1,23 +0,0 @@
-#!/bin/zsh
```

Comment:
Let's keep this. It's just a sanity check between two public repos. You can actually add that comment on top of the file.